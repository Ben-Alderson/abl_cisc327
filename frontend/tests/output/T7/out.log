Enter command: ERROR: You must login before you logout
Enter command: 