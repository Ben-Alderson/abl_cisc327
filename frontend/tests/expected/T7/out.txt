error(You must login before you can log out)


